config/crd/bases/policies.kubewarden.io_clusteradmissionpolicies.yaml

```diff
@@ -194,6 +194,10 @@ spec:
                       are ANDed.
                     type: object
                 type: object
+              policyServer:
+                default: default
+                description: PolicyServer identifies an existing PolicyServer resource.
+                type: string
```

Comment:
So this is what `+kubebuilder:default:=default` is generating, nice. I wasn't sure if `+kubebuilder:default:=default` would require a webhook, but is really nice it doesn't and uses this API machinery feature :)